clients/client-java/src/main/java/com/datastrato/gravitino/client/ErrorHandlers.java

```diff
@@ -182,9 +183,10 @@ public void accept(ErrorResponse errorResponse) {
 
         case ErrorConstants.UNSUPPORTED_OPERATION_CODE:
           throw new UnsupportedOperationException(errorMessage);
-      }
 
-      super.accept(errorResponse);
+        default:
+          throw new UnknownException(errorMessage);
+      }
```

Comment:
This changes the behavior of the existing code before it falls though to super.accept(errorResponse); I would make the default do this.

Reply:
@justinmclean Call super.accept(errorResponse) in default clause is better?

Reply:
I think so.

Reply:
> I think so.

I got it, thank you @justinmclean